packages/editor/src/components/collab-sidebar/hooks.js

```diff
@@ -372,6 +382,8 @@ export function useFloatingThread( {
 	const blockRef = useRef();
 	useBlockElementRef( thread.blockClientId, blockRef );
 
+	const blockMode = useBlockMode( thread.blockClientId );
```

Comment:
```suggestion
	const blockMode = useSelect(
		( select ) => {
			return thread.blockClientId
				? select( blockEditorStore ).getBlockMode(
						thread.blockClientId
				  )
				: null;
		},
		[ thread.blockClientId ]
	);
```

Since this hook is only used once, I think it's fine to hard-code it here.

Reply:
@t-hamano - The change has been applied and the component now uses the inline useSelect as recommended and also rebased the branch.